### DISCUSSION TITLE
Posibilities of nx-plugins

### DISCUSSION THREAD

**Discussion by AnnoDomine** [lucasvieirasilva/nx-plugins#164] [Q&A] [answered]
Hi peoples.
I have a few questions because of using this nx-plugin.

Is it possible to use it without using poetry?

Can i use it with a newer version of python (>= 3.11)?

**Comment by lucasvieirasilva** [comment, marked as answer]
Hi @AnnoDomine, 

> Is it possible to use it without using poetry?
No, all the dependency graph features are built on top of Poetry

> Can i use it with a newer version of python (>= 3.11)?
Yes
